var/spack/repos/builtin/packages/biobambam2/package.py

```diff
@@ -12,6 +12,10 @@ class Biobambam2(AutotoolsPackage):
     homepage = "https://gitlab.com/german.tischler/biobambam2"
     url = "https://gitlab.com/german.tischler/biobambam2/-/archive/2.0.177-release-20201112105453/biobambam2-2.0.177-release-20201112105453.tar.gz"
 
+    version(
+        "2.0.177-release-20201112105453",
+        sha256="ad0a418fb49a31996a105a1a275c0d1dfc8b84aa91d48fa1efb6ff4fe1e74181",
+    )  # FIXME
```

Comment:
This is the same version as below. Identical hash.

Reply:
Ah, thought I'd pulled this out. Didn't get re-added to a commit. Good catch!